velox/experimental/cudf/exec/CudfHashAggregation.cpp

```diff
@@ -799,9 +827,10 @@ RowVectorPtr CudfHashAggregation::getOutput() {
 
   // Handle partial groupby.
   if (isPartialOutput_ && !isGlobal_) {
```

Comment:
bool isPartialOutput(core::AggregationNode::Step step) {
  return step == core::AggregationNode::Step::kPartial ||
      step == core::AggregationNode::Step::kIntermediate;
}, when the companion function is count_merge_extract(step Final), it is not partial and cannot flush the final output. Please throw exception for count_merge_extract